.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: node_js
+node_js: node
+cache: yarn
+
+install:
+  - yarn install
+
+script:
+  - yarn test
+  - yarn run build
+
+deploy:
+  provider: firebase  
+  firebase-token: $FIREBASE_TOKEN   
```

Comment:
i think, we need it like this https://docs.travis-ci.com/user/deployment/firebase/

```
deploy:
  provider: firebase
  token: $FIREBASE_TOKEN 

```